### ISSUE TITLE
Daily Messages Issues

### ISSUE THREAD

**Issue by mehapps** [BlueBubblesApp/bluebubbles-server#691]
This is kind of a weird issue, my daily messages counter is currently stuck at 678, not sure where that number came from? Previously it was stuck at 1000 for all periods, but I think I left it on "past year" overnight and now it is stuck with 678. It was stuck at 1000 before (also not sure where that came from).
As I'm typing this I received/sent new messages, it moved up to 687 and I just watched it go back down to 686 lol.
I just did the calculation myself and divided total/days in a month and it should be around 1260. Attaching screenshots below of all 4 time periods
![SCR-20240902-122332@2x](https://github.com/user-attachments/assets/a81a0790-ce64-4ec4-98e9-ef57bb289cab)
![SCR-20240902-122415@2x](https://github.com/user-attachments/assets/05eca0ee-04ac-4da8-bf24-913fbb9ae16a)
![SCR-20240902-122438@2x](https://github.com/user-attachments/assets/45e63c50-ebda-4444-acdb-1e0e0cf3ac34)
![SCR-20240902-122454@2x](https://github.com/user-attachments/assets/2cac077c-fcd6-4d2b-ac74-c5581c1682c8)

**Comment by cameronaaron** [comment]
are you stil having this issue?

**Comment by mehapps** [comment]
Yeah it does look like it. Just curious, how is Daily Messages calculating done? is it the number of messages in the past 24 hours or just an average?

**Comment by zlshames** [comment]
Your daily messages widget won't change since it's not an average, but rather, just how many messages you've sent/received on the current day. Maybe I should rename it...

**Comment by zlshames** [comment]
I've updated it to say, "Messages Today" and it calculates the number of messages sent/received since the start of the day
